specification/application_layer/conventions.tex

```diff
@@ -0,0 +1,238 @@
+\section{Application-level conventions}\label{sec:application_level_conventions}
```

Comment:
This all seems pretty close to what we had before and what we've discussed in the forums. Mostly just a reminder here that this needs careful copy-editing to simplify and properly voice all the English.

Reply:
Noted.